frontend/src/router/index.ts

```diff
@@ -1,12 +1,18 @@
 import { createRouter, createWebHistory, RouteRecordRaw } from "vue-router";
 import HomeView from "../views/HomeView.vue";
+import SignUpView from "../views/SignUpView.vue";
 
 const routes: Array<RouteRecordRaw> = [
   {
     path: "/",
     name: "home",
     component: HomeView,
   },
+  {
+    path: "/auth/signup",
```

Comment:
if you don't want to have conflict in your code, we should merge this PR first and after the other two so that the routes will be right or you can change these